composites/Plugin/SnippetPreview/components/HelpTextWrapper.js

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import HelpText from "../../Shared/components/HelpText";
+import colors from "../../../../style-guide/colors.json";
+import { Button } from "../../Shared/components/Button";
+import SvgIcon from "../../Shared/components/SvgIcon";
+import { rgba } from "../../../../style-guide/helpers";
+
+
+const HelpTextContainer = styled.div`
+	max-width: 600px;
+    padding: 1 0 3 1em;
+    font-weight: normal;
+    margin: 0px 20px 10px 25px
+`;
+
+let HelpTextDiv = styled.div`
+	max-width: 400px;
+	display: block;
+	transition: all 0.5s ease;
+	overflow: hidden;
+	max-height: 0;
+`;
+
+const HelpTextButton = styled( Button )`
+	min-width: 14px;
+	min-height: 14px;
+	width: 30px;
+	height: 30px;
+	border-radius: 50%;
+	border: none;
+	clip: rect(1px 1px 1px 1px);
+	box-shadow: 0 0 0 0px #5b9dd9;
+	position: relative;
+	display: block;
+	margin: -44px -10px 10px 0px;
+	background-color: transparent;
+	float: right;
+	padding: 2px 0px 2px 6px;
+	&:hover {
+		color: ${ colors.$color_blue };
+	}
+	&:focus {
+		border: 1px solid ${ colors.$color_blue };
+		outline: none;
+		box-shadow: 0 0 3px ${ rgba( colors.$color_blue_dark, .8 ) };
+
+		svg {
+			fill: ${ colors.$color_blue };
+			color: ${ colors.$color_blue };
+		}
+	}
+`;
+
+const StyledSvg = styled( SvgIcon )`
+	vertical-align: center;
+	position: relative;
+	&:hover {
+		fill: ${ colors.$color_blue };
+	}
+`;
+
+class HelpTextWrapper extends React.Component {
+	/**
+	 * Renders the HelpTextWrapper component
+	 *
+	 * @param {Object} props The passed props.
+	 * @param {string} props.className   The class name.
```

Comment:
Either line up the documentation, or keep the seperation space between the description and the variable name the same, now it's inconsistent. And should the helpText prop also be documented here?